src/sage/geometry/polyhedron/base_QQ.py

```diff
@@ -132,7 +132,7 @@ def integral_points_count(self, verbose=False, use_Hrepresentation=False,
             sage: Q = P*(8/9)
             sage: Q.integral_points_count()
             1
-            sage: Q.integral_points_count(explicit_enumeration_threshold=0)  # optional - latte_int
+            sage: Q.integral_points_count(explicit_enumeration_threshold=0)
             1
```

Comment:
This is a removal  of a tag. Is this correct? Was it a mistake or something has changed meanwhile?

Reply:
This should not have been removed. I'll investigate.

Reply:
No, the change is actually correct. This example is handled by preprocessing and does not need `latte_int`.

Reply:
OK.